doc/sphinxext/sphinxgallery/__init__.py

```diff
@@ -1,8 +1,9 @@
 """Sphinx Gallery
 """
 import os
-__version__ = '0.0.9-dev'
```

Comment:
Ideally we should stick with a released version. @Titan-C any reason why this is not 0.0.11?